pixi.toml

```diff
@@ -87,7 +87,7 @@ scipy = "*"
 scikit-learn = "1.4.*"
 matplotlib = "<=3.8.2"
 # IO
-boto3 = "*"
+boto3 = ">=1.35.74"
```

Comment:
We could confine this to a testing dependency version requirement but I doubt it's an issue